app/containers/UIKit/VideoConferenceBlock/components/StartACallActionSheet.tsx

```diff
@@ -1,80 +1,82 @@
 import React, { useEffect, useState } from 'react';
-import { Text, View } from 'react-native';
-import Touchable from 'react-native-platform-touchable';
+import { Camera, CameraType } from 'expo-camera';
+import { View } from 'react-native';
+import { useDispatch } from 'react-redux';
 
+import { cancelCall, initVideoCall } from '../../../../actions/videoConf';
 import i18n from '../../../../i18n';
 import { getSubscriptionByRoomId } from '../../../../lib/database/services/Subscription';
 import { useAppSelector } from '../../../../lib/hooks';
 import { getRoomAvatar, getUidDirectMessage } from '../../../../lib/methods/helpers';
 import { useTheme } from '../../../../theme';
-import { useActionSheet } from '../../../ActionSheet';
 import AvatarContainer from '../../../Avatar';
 import Button from '../../../Button';
-import { CustomIcon } from '../../../CustomIcon';
-import { BUTTON_HIT_SLOP } from '../../../message/utils';
-import StatusContainer from '../../../Status';
+import { CallHeader } from '../../../VideoConf/CallHeader';
 import useStyle from './styles';
+import { ESounds, useVideoConfRinger } from '../../../../lib/hooks/useVideoConf';
 
-export default function StartACallActionSheet({ rid, initCall }: { rid: string; initCall: Function }): React.ReactElement {
+const CAM_SIZE = { height: 220, width: 148 };
+// fixed colors, do not change with theme change.
+export const gray300 = '#5f656e';
+export const gray100 = '#CBCED1';
+
+export default function StartACallActionSheet({ rid }: { rid: string }): React.ReactElement {
 	const style = useStyle();
+	const { playSound, stopSound } = useVideoConfRinger(ESounds.DIALTONE);
+
 	const { colors } = useTheme();
-	const [user, setUser] = useState({ username: '', avatar: '', uid: '' });
+	const [room, setRoom] = useState({ roomName: '', avatar: '', uid: '', direct: false });
```

Comment:
Is this a wrong merge?
We discussed this User vs Room naming a couple of weeks ago